### PR TITLE
style filterbar-dropdown

### LINKED ISSUE #91: Filterbar dropdown animation
Add animation to filter bar dropdown to create a modal effect.

### PR DESCRIPTION
**Pull request by JayDampitan**
## Changes
1. Added animations to show a modal effect on the filter bar dropdown.
2. Added a function that returns a classname based on the checkbox type. 

## Purpose
Better UI by being able to sort pokemons by their types, alphabetical order or by their values. 

## Testing Steps
1. Click on the sort Icon to make sure it gives a modal effect as it renders. 
2. Selecting checkbox types to make sure the background color changes based on the selected type of pokemon. 

Closes #91

### CODE REVIEW

File: pokeapp/src/styling/components/_filterbar.scss [hunk 1]
```diff
@@ -24,38 +24,35 @@
...
-  background-color: $component-container;
+  background-color: $secondary-color;
   position: absolute;
   top: 100%;
   right: 0;
   z-index: 30;
   border-radius: 10px;
   padding: 10px;
   overflow: hidden;
+  //     border: 1px solid $pokedollar-color;
```
Comment (Lap343) [inline pokeapp/src/styling/components/_filterbar.scss:35]: Do you still need this comment?

File: pokeapp/src/styling/components/_filterbar.scss [hunk 2]
```diff
@@ -79,7 +76,60 @@
 //---------------------------------------------------------------DROPDOWN ANIMATION END
 
 .active {
-  background-color: red;
+  &-Fire {
```
Comment (Lap343) [inline pokeapp/src/styling/components/_filterbar.scss:79]: nice using sass for rendering the different types

File: pokeapp/src/styling/components/_filterbar.scss (hunk 1 above)
Comment (BlackCubes) [inline pokeapp/src/styling/components/_filterbar.scss:35]: Muh man! Dis comment! 😘

Comment (BlackCubes) [reply pokeapp/src/styling/components/_filterbar.scss:35]: Maybe remove?

**Review by connielion — APPROVED** [review]
Looks good! As Luis said, do you still need that comment in `_filterbar.scss`? If not, remove and push again :).

File: pokeapp/src/styling/components/dist/_filterbar.css [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+.filter-bar-dropdown {
+  transform: scaleY(0);
+  overflow: hidden;
+  opacity: 0;
+}
+
+.filter-bar-dropdown-active {
+  transform-origin: top right;
+  animation: 0.7s ease 0s 1 normal forwards running filter-dropdown-active;
```
Comment (BlackCubes) [inline pokeapp/src/styling/components/dist/_filterbar.css:39]: Can't wait to see this

File: pokeapp/src/styling/components/_filterbar.scss [hunk 4]
```diff
@@ -79,7 +76,60 @@
...
+  }
+  &-Rock {
+    background-color: $rock;
+  }
+  &-Flying {
+    background-color: $flying;
+  }
+  &-Ground {
+    background-color: $ground;
+  }
```
Comment (JorgeLVilla) [inline pokeapp/src/styling/components/_filterbar.scss:132]: Nice organization.

**Review by BlackCubes — APPROVED** [review]
Looks noice! Good job!

**Review by Lap343 — APPROVED** [review]
Good code. Mostly CSS. Just suggesting to look at the one line comment and if needed then all is ok

**Review by JorgeLVilla — APPROVED** [review]
Nice job!!

**Pull request merged**